### PR TITLE
WIP: Automation mysql installation Issues#12

### PR DESCRIPTION
**Pull request by BarakMerdler**
issue #16 
Implement a bash script which install mysql on a fedora 30 VM

first try to add script.sh

### CODE REVIEW

File: script.sh [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+
+sudo dnf update
```
Comment (lmilbaum) [inline script.sh]: You do not need to add sudo on every command. You can run the script with sudo. That should do.

File: script.sh [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+sudo dnf install https://repo.mysql.com//mysql80-community-release-fc30-1.noarch.rpm
+sudo dnf install mysql-community-server
+
+systemctl start mysqld
```
Comment (lmilbaum) [inline script.sh]: How do you verify the service is persistent between restarts?

**Review by lmilbaum — CHANGES_REQUESTED** [review]
The name of the PR should reflect it content. Please update it.
And, mention the issue number this PR relates to.

**Comment by lmilbaum** [comment]
you have two commits with the same name. squash them into one.

File: script.sh (hunk 2 above)
Comment (lmilbaum) [reply script.sh]: When you do changes following a review comment, do add your feedback in the conversation.

Comment (lmilbaum) [reply script.sh]: why did you remove line 8?

Comment (BarakMerdler) [reply script.sh]: got it, working on it

Comment (BarakMerdler) [reply script.sh]: worked on the file, submitted chenges

**Comment by lmilbaum** [comment]
@BarakMerdler As long as you work on this PR, please add the following prefix to your PR subject: "WIP:"

**Review by lmilbaum — CHANGES_REQUESTED** [review]
Rebase your PR with latest changes in upstream

**Comment by lmilbaum** [comment]
@BarakMerdler First problem is the link to the issue. No need to mention the issue # in the PR name.
You should specify it in (usually) the first comment. For now, you added a link to itself.

File: script.sh (hunk 2 above)
Comment (lmilbaum) [reply script.sh]: I don't see where you addressed the persistance issue. Have you tested it?

**Comment by lmilbaum** [comment]
I am totally lost of this PR.
First issue - the WIP prefix in the PR name states that it is not ready for review. Is it?
Second issue - no link to the issue
Third issue - @Nomasson added few commits which I am not sure if related. A PR should include ONLY related files to the PR mission
Forth issue - too many commits. A PR should have only one/two commits. This one has 19!!!!

**Review by lmilbaum — CHANGES_REQUESTED** [review]

**Pull request closed without merging**